### PR TITLE
WIP: Add QRCode in Api clients management for each clients

### PR DESCRIPTION
**Pull request by bourvill**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | no
| Translation   | needed
| Fixed tickets |
| License       | MIT

Add QRCode in Api clients management for each clients

**Help needed for** 
- [ ] display/hide the QRCode
- [ ] translations

### CODE REVIEW

File: src/Wallabag/CoreBundle/Resources/translations/messages.en.yml [hunk 1]
```diff
@@ -452,6 +452,7 @@ developer:
         field_uris: 'Redirect URIs'
         field_grant_types: 'Grant type allowed'
         no_client: 'No client yet.'
+        show_qrcode_iphone: 'Show QRCode iphone'
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.en.yml]: i**P**hone please

**Comment by bourvill** [comment]
@j0k3r it's done

File: src/Wallabag/CoreBundle/Resources/translations/messages.fr.yml [hunk 2]
```diff
@@ -452,6 +452,7 @@ developer:
         field_uris: "Adresse de redirection"
         field_grant_types: "Type de privilège accordé"
         no_client: "Aucun client pour le moment"
+        show_qrcode_iPhone: 'Voir le QRCode iphone'
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/translations/messages.fr.yml]: I mean, the translation should be i**P**hone, not the key 😄

Comment (bourvill) [inline src/Wallabag/CoreBundle/Resources/translations/messages.fr.yml]: lol, done

**Comment by j0k3r** [comment]
@wallabag/translators hey guys, can you drop us a comment about translating `Show iPhone's QRCode` in your language? Thanks 👊

**Comment by Quent-in** [comment]
In OC: Mostrar lo còdi QR per iPhone

**Comment by mruminski** [comment]
In PL: Pokaż kod  QR iPhone'a

**Comment by matteocoder** [comment]
Is the QR code meant to be scanned by the iPhone? In that case the Italian translation is: mostra codice QR per iPhone.

**Comment by jlnostr** [comment]
Can someone explain this feature to me? Why is it limited to the iPhone?

**Comment by bourvill** [comment]
Hello @jlnostr, with this new QRCode, more info is transmitted to the app.

We provide username, url of instance, clientId and clientSecret. 
You can easily use in other app

**Comment by jlnostr** [comment]
@bourvill In this case, I would drop the limitiation to the iPhone in the translation. Maybe there should be a sidenote, that this feature is currently only supported on iOS. And to be more flexible, it should be separated from the application, but should be shown in the docs IMO.

What do you think about this @wallabag/core?

**Comment by bourvill** [comment]
@jlnostr i'm agree.

I think i'll rename the scheme wallabagv2 -> wallabag-token ?

**Comment by nicosomb** [comment]
> I would drop the limitiation to the iPhone in the translation

OK for me.

**Comment by tcitworld** [comment]
You may want https://github.com/wallabag/wallabag/pull/3068 before

**Pull request closed without merging**